spec/components/placement/summary_component_spec.rb

```diff
@@ -22,7 +22,7 @@
     )
   end
   let(:placement_1) { create(:placement, school:, subjects:, mentors:) }
-  let(:placement_2) { create(:placement, school:, subjects: [subject_2], mentors:) }
+  let(:placement_2) { create(:placement, school:, subjects: [subject_2], mentors:, provider: create(:placements_provider)) }
```

Comment:
I expect this is more performant (albeit _only very slightly_). The provider will still be created when the overall placement is created.

```suggestion
  let(:placement_2) { create(:placement, school:, subjects: [subject_2], mentors:, provider: build(:placements_provider)) }
```